packages/builders/src/components/Component.ts

```diff
@@ -16,30 +11,13 @@ export type AnyAPIActionRowComponent = APIActionRowComponent<APIActionRowCompone
  *
  * @typeParam DataType - The type of internal API data that is stored within the component
```

Comment:
Something like this?

```suggestion
 * @typeParam Component - The type of API data that is stored within the builder
```